Tests/WebSocketKitTests/Utilities.swift

```diff
@@ -169,29 +169,29 @@ internal final class WebsocketBin {
     // In the TLS case we must set up the 'proxy' and the 'server' handlers sequentially
     // rather than re-using parts because the requestDecoder stops parsing after a CONNECT request
     private func syncAddTLSHTTPProxyHandlers(
-        to channel: Channel,
+        to channel: any Channel,
         proxyConfig: ProxyConfig,
         expectedAuthorization: String?,
-        upgraders: [HTTPServerProtocolUpgrader]
+        upgraders: [any HTTPServerProtocolUpgrader]
     ) throws {
         let sync = channel.pipeline.syncOperations
         let promise = channel.eventLoop.makePromise(of: Void.self)
 
-        let responseEncoder = HTTPResponseEncoder()
-        let requestDecoder = ByteToMessageHandler(HTTPRequestDecoder(leftOverBytesStrategy: .forwardBytes))
-        let proxySimulator = HTTPProxySimulator(promise: promise, config: proxyConfig, expectedAuthorization: expectedAuthorization)
+        let responseEncoder = NIOLoopBound(HTTPResponseEncoder(), eventLoop: channel.eventLoop)
```

Comment:
It's tests so doesn't matter as much but why the loop bound dance here? We're `@unchecked Sendable` and `@preconcurrency` importing the types. I would have assumed one of the 3 approaches would be enough

Reply:
Because futures are now Strict Concurrency-correct and _everything_ you capture in a closure has to be `Sendable` in ways that `@preconcurrency` would only silence if you applied it to the entire `NIOCore` import.